src/InOneWeekend/camera.h

```diff
@@ -85,10 +84,9 @@ class camera {
         lens_radius = aperture / 2;
     }
 
-    ray get_ray(double s, double t) const {
-        // Return the ray from the projection point to the indicated pixel. Coordinates s,t are
-        // the normalized image-based coordinates of the pixel. Image left is s=0, image right
-        // is s=1, image top is t=0, image bottom is t=1.
```

Comment:
What was the decision behind removing the comments?

Reply:
This is one of several increments to follow, so if there will be a comment, it will come after the dust settles.